src/gen-ts.ts

```diff
@@ -234,6 +245,63 @@ function handleFunction(feature: AnalyzerFunction, root: ts.Document) {
 }
 
 /**
+ * Convert the given Analyzer properties to their TypeScript declaration
+ * equivalent.
+ */
+function handleProperties(analyzerProperties: Iterable<analyzer.Property>):
+    ts.Property[] {
+  const tsProperties = new Array<ts.Property>();
+  for (const property of analyzerProperties) {
+    if (property.inheritedFrom) {
+      continue;
+    }
+    tsProperties.push({
+      kind: 'property',
+      name: property.name || '',
+      description: property.description || '',
+      type: property.type ? closureTypeToTypeScript(property.type) : ts.anyType,
```

Comment:
seems like this conditional could just be in `closureTypeToTypeScript`

Reply:
Done.